src/conversation.js

```diff
@@ -25,7 +26,7 @@ const shutdown = (bot, message) => {
       {
         pattern: bot.utterances.yes,
         callback: function(response, convo) {
-          convo.say("Alright Sir, if you'll not be needing me, I'll close down for awhile");
+          convo.say('Alright Sir, if you\'ll not be needing me, I\'ll close down for awhile');
```

Comment:
I'd prefer back to double-quotes over a template string just to avoid escaping. ESLint has a config option that allows "single quotes all the time, unless it's to avoid escaping in which case double quotes are OK".

Alternatively, you could standardize on double quotes everywhere. Whatevs, really.